specs/latest/2.0/index.html

```diff
@@ -688,8 +688,8 @@ <h2 class="no-toc">Table of contents</h2>
   /* Buffer objects */
   // WebGL1:
   void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
-  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage);
```

Comment:
@jdashg This actually doesn't fit with WebIDL. 

Looking at https://heycam.github.io/webidl/#idl-overloading the matching table, numeric types / interface is no longer distinguishable because interface is no longer nullable.